### ISSUE TITLE
CI: Failed to load /var/run/cilium/state/bpf_sockops.o

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#8023]
The following log was found in a CI run:
```
pod-kube-system-cilium-9h7g5-cilium-agent-previous.log:2019-05-14T17:23:48.664782002Z level=error msg="Failed to load /var/run/cilium/state/bpf_sockops.o: exit status 255: libbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"CALLS_MAP\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"CALLS_MAP\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_encrypt_state\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_ep_to_policy\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_events\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_ipcache\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb4_backends\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb4_reverse_nat\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb4_rr_seq\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb4_rr_seq_v2\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb4_services\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb4_services_v2\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb6_backends\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb6_reverse_nat\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb6_rr_seq\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb6_rr_seq_v2\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb6_services\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lb6_services_v2\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_lxc\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_metrics\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_policy\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_policy\" has unrecognized, non-zero options\nlibbpf: maps section in /var/run/cilium/state/bpf_sockops.o: \"cilium_sock_ops\" has unrecognized, non-zero options\nlibbpf: Program '.text' contains unrecognized relo data pointing to section 0\nError: failed to open object file\n" subsys=sockops
```

**Comment by tgraf** [comment]
Logs from CI run
[5dbd0945_K8sDatapathConfig_Encapsulation_Check_connectivity_with_sockops_and_VXLAN_encapsulation.zip](https://github.com/cilium/cilium/files/3179507/5dbd0945_K8sDatapathConfig_Encapsulation_Check_connectivity_with_sockops_and_VXLAN_encapsulation.zip)

**Comment by jrfastab** [comment]
Fix here. But that error should be harmless and test should continue running normally.

https://github.com/cilium/cilium/pull/8022

**Comment by tgraf** [comment]
Fixed by #8022
